Kitodo/src/test/java/test/SimpleLoginST.java

```diff
@@ -0,0 +1,411 @@
+        } catch (IOException e) {
+            logger.error(e.getMessage());
+        }
+        return screenshotFile;
+    }
+
+    private static void setExecutePermission(File file) throws IOException {
+        Set<PosixFilePermission> perms = new HashSet<>();
+        perms.add(PosixFilePermission.OWNER_READ);
+        perms.add(PosixFilePermission.OWNER_WRITE);
+        perms.add(PosixFilePermission.OWNER_EXECUTE);
+
+        perms.add(PosixFilePermission.OTHERS_READ);
+        perms.add(PosixFilePermission.OTHERS_WRITE);
+        perms.add(PosixFilePermission.OTHERS_EXECUTE);
+
+        perms.add(PosixFilePermission.GROUP_READ);
+        perms.add(PosixFilePermission.GROUP_WRITE);
+        perms.add(PosixFilePermission.GROUP_EXECUTE);
+
+        Files.setPosixFilePermissions(file.toPath(), perms);
+    }
+
+    private static void extractZipFileToFolder(File zipFile, File destinationFolder) {
+        try {
+            ZipFile zip = new ZipFile(zipFile);
+
+            destinationFolder.mkdir();
+            Enumeration zipFileEntries = zip.entries();
+
+            while (zipFileEntries.hasMoreElements()) {
+                ZipEntry entry = (ZipEntry) zipFileEntries.nextElement();
+                String currentEntry = entry.getName();
+
+                File destFile = new File(destinationFolder, currentEntry);
+                File destinationParent = destFile.getParentFile();
+
+                destinationParent.mkdirs();
+
+                if (!entry.isDirectory()) {
+                    BufferedInputStream bufferedInputStream = new BufferedInputStream(zip.getInputStream(entry));
+                    int currentByte;
+                    byte data[] = new byte[BUFFER_SIZE];
+
+                    FileOutputStream fileOutputStream = new FileOutputStream(destFile);
+                    BufferedOutputStream bufferedOutputStream = new BufferedOutputStream(fileOutputStream, BUFFER_SIZE);
+
+                    while ((currentByte = bufferedInputStream.read(data, 0, BUFFER_SIZE)) != -1) {
+                        bufferedOutputStream.write(data, 0, currentByte);
+                    }
+                    bufferedOutputStream.flush();
+                    bufferedOutputStream.close();
+                    bufferedInputStream.close();
+                }
+            }
+        } catch (Exception e) {
+            logger.error(e.getMessage());
+        }
+    }
+
+    /**
+     * Unpack the archive to specified directory.
+     *
+     * @param file
+     *            The tar or tar.gz file.
+     * @param outputDir
+     *            The destination directory.
+     * @param isGZipped
+     *            True if the file is gzipped.
+     */
+    private static void extractTarFileToFolder(File file, File outputDir, boolean isGZipped) {
+        FileInputStream fileInputStream = null;
+        TarInputStream tarArchiveInputStream = null;
+        try {
+            fileInputStream = new FileInputStream(file);
```

Comment:
Same here for `fileInputStream` and `tarArchiveInputStream`: use the [try-with-resources-statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html).